frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import { Grid } from "@material-ui/core";
+import { ChoroplethMap } from "../../charts/ChoroplethMap";
+import { Fips } from "../../data/utils/Fips";
+import { Legend } from "../../charts/Legend";
+import { MetricConfig } from "../../data/config/MetricConfig";
+import { Row, FieldRange } from "../../data/utils/DatasetTypes";
+import styles from "./MultiMapDialog.module.scss";
+
+export interface MultiMapDialogProps {
+  metricConfig: MetricConfig;
+  breakdown: string;
+  fips: Fips;
+  data: Row[];
+  fieldRange: FieldRange | undefined;
+  open: boolean;
+  handleClose: () => void;
+  breakdownValues: string[];
+}
+
+export function MultiMapDialog(props: MultiMapDialogProps) {
+  return (
+    <Dialog
+      open={props.open}
+      onClose={props.handleClose}
+      maxWidth={false}
+      scroll="paper"
+      aria-labelledby="Dialog showing choropleth maps of each breakdown category with the same scale."
+    >
+      <DialogContent dividers={true}>
+        <Grid container justify="space-around">
+          <Grid item xs={6}>
+            <Alert severity="info">
+              This scale is a{" "}
+              <a href="https://vega.github.io/vega/docs/scales/#quantile">
+                quantile
```

Comment:
I'd pull "quantile" into a variable and use that throughout this class so if it were to change it's easy to just update the variable value